src/components/Home/Home.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import './Home.css';
-import HomeHandsSlot from './assets/HandsSlot.svg';
-import HomeHandsRight from './assets/HandsRight.svg';
-import HomeHandsCenter from './assets/HandsCenter.svg';
-import HomeHandsLeft from './assets/HandsLeft.svg';
+import HomeHandsSlot from './../../utils/assets/hands/HandsSlot.svg';
+import HomeHandsRight from './../../utils/assets/hands/HandsRight.svg';
+import HomeHandsCenter from './../../utils/assets/hands/HandsCenter.svg';
+import HomeHandsLeft from './../../utils/assets/hands/HandsLeft.svg';
```

Comment:
With the SVGs being statis files would it not make sense to move them to /public along with the fonts?

Reply:
I think that would be logical personally.

Reply:
Will leave this for another PR